wbooks/src/app/screens/BookList/index.tsx

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-use-before-define */
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+
+import BookItem from './components/BookItem';
+
+const BookList = () => (
+  <View style={styles.container}>
+    <BookItem />
+    <BookItem />
+    <BookItem />
+    <BookItem />
```

Comment:
Pass data via props, use `mockBook` (for example: `mockBook[0]`, `mockBook[1]`, etc...)

Reply:
Replaced with temporal .map